### PR TITLE
Add ContentType header to Update-GitHubIssue

### PR DESCRIPTION
**Pull request by Alancere**
image

This PR is to fix the issue 
https://github.com/Azure/azure-sdk-tools/issues/7621#issue-2114362657

### CODE REVIEW

File: eng/common/scripts/Invoke-GitHubAPI.ps1 [hunk 1]
```diff
@@ -403,6 +403,7 @@ function Update-GitHubIssue {
           -Uri $uri `
           -Headers (Get-GitHubApiHeaders -token $AuthToken) `
           -MaximumRetryCount 3
+          -ContentType "application/x-www-form-urlencoded"
```
Comment (raych1) [inline eng/common/scripts/Invoke-GitHubAPI.ps1:406]: Can we use `application/json` as the body is converted to JSON in line#402?

**Review by weshaggard — APPROVED** [review]

**Review by danieljurek — CHANGES_REQUESTED** [review]
The current syntax will not pick up the `-ContentType` parameter because the script does not have a backtick to continue parsing to the next line.

**Comment by weshaggard** [comment]
@raych1 Lets please be sure to test this change.

**Review by danieljurek — APPROVED** [review]

**Comment by raych1** [comment]
> @raych1 Lets please be sure to test this change.

@weshaggard , this is the [test pipeline run](https://dev.azure.com/azure-sdk/internal/_build/results?buildId=3471789&view=logs&j=33796ffc-d0c5-548c-0c1d-9d1ddb84594a&t=49452e74-a04a-59c1-118d-9b0749e7229c)

**Comment by weshaggard** [comment]
@raych1 I don't suspect that is hitting this code path on PS7.4. Just please be sure to check the same pipeline that was failing in the go repo with this fix.

**Comment by raych1** [comment]
> @raych1 I don't suspect that is hitting this code path on PS7.4. Just please be sure to check the same pipeline that was failing in the go repo with this fix.

@weshaggard , the failure was caught in the Go release pipeline, and we will verify it during this month's release.

**Pull request merged**